frontend/src/metabase-types/api/card.ts

```diff
@@ -113,14 +113,44 @@ export type SeriesOrderSetting = {
   color?: string;
 };
 
-export type ColumnFormattingSetting = {
-  columns: string[]; // column names
-  color?: string;
-  type?: string;
-  operator?: string;
-  value?: string | number;
-  highlight_row?: boolean;
+export type CommonOperator = "is-null" | "not-null";
```

Comment:
Let's add some prefix so that it's clear that it's an operator for formatting only. E.g. `StringFormattingOperator.` Real MBQL operators are a bit different, so please do not make these types generic.

Reply:
I agree, these operators specific to conditional formatting so it is better to prefix them with `ConditionalFormatting`